internal/cluster/cluster.go

```diff
@@ -826,31 +826,39 @@ func (m *Manager) SetConnectivityMajorityGroupsForCluster(clusterID strfmt.UUID,
 	return nil
 }
 
-func (m *Manager) DeleteClusterLogs(ctx context.Context, c *common.Cluster, objectHandler s3wrapper.API) error {
+func (m *Manager) deleteClusterFiles(ctx context.Context, c *common.Cluster, objectHandler s3wrapper.API, folder, desc string) error {
 	log := logutil.FromContext(ctx, m.log)
-	files, err := objectHandler.ListObjectsByPrefix(ctx, fmt.Sprintf("%s/logs/", c.ID))
+	files, err := objectHandler.ListObjectsByPrefix(ctx, fmt.Sprintf("%s/%s/", c.ID, folder))
 	if err != nil {
 		return common.NewApiError(http.StatusNotFound, err)
 	}
 
 	var failedToDelete []string
 	for _, file := range files {
-		log.Debugf("Deleting cluster %s S3 log file: %s", c.ID.String(), file)
+		log.Debugf("Deleting cluster %s S3 %s file: %s", c.ID.String(), file, desc)
```

Comment:
These params are backwards, but generally I feel like it's overkill to send in a separate description for the files being deleted.
The object (file in this case) should have the prefix so I think we'll be fine removing `desc` entirely.